ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -344,6 +344,37 @@ func accountsCreateBoxTable(ctx context.Context, tx *sql.Tx) error {
 	return nil
 }
 
+// performkvstoreNullBlobConversion scans keys with null blob value, and convert the value to `[]byte{}`.
+func performkvstoreNullBlobConversion(ctx context.Context, tx *sql.Tx) (err error) {
```

Comment:
Since the two defers are not needed to update `err`, I'd prefer not using the named return value, and the naked `returns` become `return err`.

Reply:
see [per PR review comments, renaming method and removing db close](https://github.com/algorand/go-algorand/pull/5225/commits/3739d3f7cfeeac0e6269c5a27483a970ac756bff)